### PR TITLE
Create config for Wurm Unlimited

### PR DESCRIPTION
**Pull request by smalls89**
Config xml for Wurm Unlimited

### CODE REVIEW

File: modules/config_games/server_configs/wurmu_win64.xml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    <mod key="wurmu">
+      <name>none</name>
+      <installer_name>402370</installer_name>
+      <installer_login>anonymous</installer_login>
+    </mod>
+  </mods>
+  <server_params>
+    <param id="SERVERNAME" key="servername=" type="text">
+      <option>ns</option>
+      <desc>The name of the server that will be visible in server browsers. (NOTE: If the server name contains spaces, quotes are required! ie. "Server Name")</desc>
```
Comment (Zorrototo) [inline modules/config_games/server_configs/wurmu_win64.xml]: you can use different options to do what you want, I think it is:

        nothing / no value = placed as is
        s = space / separated
        q = quoted
        sq = space and quotes
        sc = space and ends with a comma
        sqc = space, quoted, and ends with a comma

Comment (Zorrototo) [inline modules/config_games/server_configs/wurmu_win64.xml]: Just my two cents :)

**Comment by rocco27** [comment]
2 more things to make your xml nicer:
- Please do not set custom game name, follow the original game name. Example: Wurm Unlimited Server Windows is total wrong and unnecessery, use only Wurm, it is enought.
- You can set the hostname automaticly and we do this usually. The owner always set the hostname in the OGP, why we force them to do it twice?

**Comment by own3mall** [comment]
Please make the changes recommended above, update your branch and pull request, and then I'll merge it.  Thanks!

File: modules/config_games/server_configs/wurmu_win64.xml (hunk 1 above)
Comment (smalls89) [inline modules/config_games/server_configs/wurmu_win64.xml]: True but i tried various options and i couldn't get it to work correctly, the cli param needs to have quotes if it has a space but also it cant have a separating space ie (servername="server Name")

this was the only way i could get it to work, i might have overlooked something though...

**Comment by smalls89** [comment]
rocco
> Same with the admin password, this called RCOn generally and OGP can manage it
Could not find a correct query type? Why is it missing?

Im assuming these are refering to rcon in the way most games use it... Wurm does not have such a feature

the admin password is to be used ingame via the console

else there is the prompt window of the server, but you can't type in it...

**Comment by rocco27** [comment]
Doesn't matter how can you manage RCON/Admin ingame, when I said OGP can manage it I thought you can set the password automaticly in the start up command line by OGP. If the game server support general RCON then it is other thing and maybe you can manage the RCON comands in the Log window from console.

**Comment by rocco27** [comment]
Doesn't matter how can you manage RCON/Admin ingame, when I said OGP can manage it I thought you can set the password automaticly in the start up command line by OGP. If the game server support general RCON then it is other thing and maybe you can manage the RCON comands in the Log window from console

**Comment by smalls89** [comment]
I image these ports can be set to anything, but these are defaults

Wurm is unique when i comes to admin, everything is done through an in game item oe in game console, there is NO external console prompt, rcon, or otherwise

**Comment by Zorrototo** [comment]
Just a question, why do you have this post install? is it mandatory when you install a fresh server? The post install script is used after the Install or update of server.

**Comment by smalls89** [comment]
I saw no alternative as the install puts down the 2 backup maps. If the user chooses to use the default maps, the server will throw the biggest fit about the maps not being found.


Side note with using q option, CLI separates it with a space, don't know if this is a bug/glitch, out I'm just doing it wrong...

**Comment by Zorrototo** [comment]
You mean the "q" option for servername in server parameters is not working for you? be more specific to speed up the polishing of your xml :)

**Comment by own3mall** [comment]
If you look at https://github.com/OpenGamePanel/OGP-Website/issues/19 there is only ns... adding new options now.

**Comment by own3mall** [comment]
Hope I didn't break anything.  I documented the changes:

https://github.com/OpenGamePanel/OGP-Website/wiki/XML-Notes

If any of you want to help complete the XML documentation following a format similar to the one in place (unless you can come up with a better one), I'll grant you access.  We can add to it as confusion and clarification is needed.

**Comment by smalls89** [comment]
so option q should work now???

**Comment by Zorrototo** [comment]
as I said 

> edit4: also your VAC parameter seems to be not valid for Wurm Unlimited absolutely no reference in official doc, and even the log tells you it is wrong.

`Unknown parameter: -insecure`

**Comment by own3mall** [comment]
Yes, "q" should work.

**Comment by Zorrototo** [comment]
@own3mall if you can create a sub page in the wiki like "WIKI WIP" where I can import the current one and start complete it, test things and all (I'm new to GitHub and don't want to break anything on the real one :D) and grant me access to this sub-page, I'll start doing it. When I'll have it complete, polished and verified we can then import it in the real one and destroy this 'WIKI WIP" page.

**Comment by smalls89** [comment]
I suppose this is done.... Unless there is a way to edit sqlite, all the remaining settings are in a sqlite db within the respective map folder...

**Comment by Zorrototo** [comment]
Seems pretty good yes. Don't have the game to test but server starts good.

Can you change it back to win32 please, this way we keep the good organisation for ogp xml :D (some older xml don't  but for now better use the good naming convention)

**Comment by Zorrototo** [comment]
Something you can also add is an icon for this game server, I think after that this can be accepted as a functional XML :) 👍

**Comment by smalls89** [comment]
ih, i thought the win32 and win64 where for specific versions, wurm is indifferent, it works on either, and depends on the version of java installed locally, can change it back if it is still an issue...

**Comment by smalls89** [comment]
idk how the icons work....

**Comment by Zorrototo** [comment]
For your XML simply upload the icon to OGP Panel folder /images/icons/ name should match XML so wurmu.png will work.

**Comment by own3mall** [comment]
Yay for making progress!  Let me know when this is ready.

**Comment by smalls89** [comment]
believe thats about it...

**Pull request merged**